### ISSUE TITLE
Next feature release

### ISSUE THREAD

**Issue by kumy** [cgeo/cgeo#5637]
#ToDo for a possible feature release:

- [x] #5536 Performance issue
- [x] #5630 Caches from EC shown as saved once opened
- [x] #5629 Reuse string for "Create account" in new authentication screens
- [x] #5623 Long press menu - problem with long strings
- [x] #5600 Cause string missing on login failure
- [x] #5636 Changed Waypoint Title overwritten on Refresh
- [x] #5550 Checkbox based list selection
- [x] #5646 Cache not stored if setting for list selection is disabled 
- [x] #5701 Manage list button not working
- [x] #5796 Improve debug log mode warning
- [x] #5797 Better strings for "New Map"
- [x] #5873 Change List handling buttons for #5839
- [x] https://github.com/cgeo/cgeo.github.io/issues/96

**Comment by kumy** [comment]
@Lineflyer I've opened this issue to not forget. Please amend it as you wish :)

**Comment by kumy** [comment]
add #5636

**Comment by Lineflyer** [comment]
From my perspective I don't think the new multi-list implementation is ready.
So IMHO implementing something like #5550 or even going into direction of #5540 would be good before releasing this new feature.

**Comment by Lineflyer** [comment]
I would like to add #5550 to this list as I think the current multi list is not yet ready from the UI perspective.

**Comment by Lineflyer** [comment]
Added #5701

**Comment by Lineflyer** [comment]
I think we have come to a quite stable condition for a next possible feature release. Only some issues still open as listed above.

But the question would be:
How to deal with the new map implementation: Hide it for now or dare to replace the old before release?

AFAICS the found bugs have been fixed (only one which is currently open) and it looks rather stable to me. 
Opinions?

**Comment by rsudev** [comment]
Regarding the new map implementation: IMO it cannot yet replace the existing map, because it is mapsforge-offline only. I don't think we need to hide it, as it can live it's 'experimental' life as intended, we might consider a different name perhaps, before going puplic.
Besides the noted issue (#5759) I think there are some more smaller features missing (circles, show as list, ...). Stability wise I think it is quite ok, though.

**Comment by Lineflyer** [comment]
@rsudev:
Agree!
So we would keep it as experimental for the next feature release, can rework and finalize the feature set afterwards and probably have it replace the existing mapsforge with the feature release after the next.
Would you open issues for the feature regressions to track it (i.e. features of the current implementation which are missing in the new one (e.g. circles))?

**Comment by Bananeweizen** [comment]
I know @rsudev is already on vacation for the next 2 weeks. I will go on programming-less vacation Saturday, 2016-07-30 for a week. Over the last weeks quite some bug fixes, memory leak fixes and upgrades of dependencies happened. Not sure what your schedules look like, but end of July looks like a perfect split to the release branch for me, for some more testing and releasing. I will try to only do further cleanups the next 2 days.

**Comment by samueltardieu** [comment]
Looks fine to me too.

**Comment by Lineflyer** [comment]
I agree to @Bananeweizen for branching.
But due to the fact that we have huge differences between current `master` and `release` and the fact that due to private issues my testing time is quite limited I suggest (if technically possible) to branch now but at least save the current `release` code somewhere (another branch) for an emergeny solution in case we still have regressions after branching but have to release immediately.

**Comment by Lineflyer** [comment]
From IRC:
```
[23:29] <Lineflyer> Hello!
[23:29] <Lineflyer> As written in https://github.com/cgeo/cgeo/issues/5637#issuecomment-236713393 I would prefer branching now as...
[23:29] <Lineflyer> 1. master is quite stable
[23:30] <Lineflyer> 2. we get a base for testing a new release
[23:30] <Lineflyer> 3. I might need quite some time for testing and want to avoid continous changes in that period
[23:31] <Lineflyer> The only proposal given the current situation: Can we freeze the current release branch somewhere as I expect a long trial before release and we might be forced to release short termed.
[23:31] <Lineflyer> That is mainly caused by the long period without feature release and to my limited amount of time for testing at the moment :(
[23:32] <Lineflyer> Please comment on #5637
[23:32] <cgeo-irct> #5637 [open] Next feature release <https://github.com/cgeo/cgeo/issues/5637>
[23:33] <Lineflyer> Good night
```

**Comment by samueltardieu** [comment]
I'll take care of branching and creating `old-release` from the current `release`.

**Comment by samueltardieu** [comment]
Branching done and milestone changed for fixed issues and release planification.

**Comment by Lineflyer** [comment]
Great @samueltardieu 

Now everyone should take care, that late incoming fixes/improvements for new features might be meaningful to be based on `release`

**Comment by Lineflyer** [comment]
Removed #5638 Prompt user when unassigning cache from last list? from the ToDos (see comment in this issue)

**Comment by Lineflyer** [comment]
btw @samueltardieu  Are all these branches needed anymore?
I gues those `issue` branches might be oudated as well as the `cgeo-legacy` which was depublished quite some months ago. Maybe you want to clean up?

**Comment by samueltardieu** [comment]
I've removed the `issue-`* branch as the corresponding issues have been closed. We can keep `bb10` and `cgeo-legacy` for historical purpose.

**Comment by Lineflyer** [comment]
I think the RC behaves really good and until now (still need to check support mails tonight) we did not get any major negative feedback.

From my perspective it would make sense to optimize the sorting of the new icons as suggested in #5893 before release if you agree.

**Comment by Lineflyer** [comment]
Is there any fix which shall be included on 'release'?

**Comment by Lineflyer** [comment]
I prepared the release and have built it.
Some translations might be not included (catalan, spanish addons) but we can still do that later on (with next bugfix release or replace staged release before going to 100%).
I will publish 2016.08.14 to 20% of users in the next minutes.

- [ ] Tag release on Github
- [ ] Publish to Fdroid
- [ ] Communicate on social networks (once it is available)
- [ ] Recheck milestones on Github (if not already done)

**Comment by kumy** [comment]
Nice work!
Btw release changelog is missing #4865 "Show logbook of GeoKrety trackables"

**Comment by Lineflyer** [comment]
We do now have approx. 5% user on new release and no critical or mass crash reports.
Lets wait one more day and then decide on going to 100%.

**Comment by Bananeweizen** [comment]
@kumy that's why it is best to include a change of changelog_master/changelog_release.xml directly in every bugfix commit. I also have a hard time remembering what needs to be added otherwise before starting a release.

**Comment by Lineflyer** [comment]
FYI:
2016.08.14 has been replaced by 2016.08.16 and staged to 50% for now.
Remaining items as listed above are still open.

Lets wait 1-2 days and then go to 100% if no more critical issues arise.

**Comment by Lineflyer** [comment]
Release 2016.08.16 set to 100%

**Comment by Lineflyer** [comment]
Release is available. FB is posted, G+ not working for me.
Notification sent.

So we are done!! Thanks to all!!
